frontend/amundsen_application/static/js/components/LineageGraph/styles.scss

```diff
@@ -0,0 +1,82 @@
+@import 'variables';
+@import 'typography';
+
+$svg-label-width: 150px;
+$svg-label-height: 30px;
+$node-color-fill: #fff;
+$node-color-outline: $purple60;
+$node-stroke-width: 3px;
+$svg-link-color-outline: rgba(12, 11, 49, 0.16);
+$svg-link-stroke-width: 2px;
+
+.lineage-graph {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  //padding-bottom: 100%;
+  vertical-align: top;
+  overflow: hidden;
+  background: linear-gradient(90deg, $gray5 50%, $gray0 50%);
+
+  .svg-content {
+    display: block;
+    margin: auto;
+    //position: absolute;
+    //top: 0;
+    //left: 0;
+
+    .direction-label {
+      font-family: $font-family-header !important;
+      font-weight: $font-weight-header-bold;
+      color: $gray30;
+      width: $svg-label-width;
+      height: $svg-label-height;
+      text-transform: uppercase;
```

Comment:
Is there a way of using our new standard font styles (https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/static/css/_typography-default.scss#L17)

Also, I think we will move out of using uppercase for a11y reasons. Probably the designs are not reflecting this, but it was something new that we noticed later.